interfaces/webusb.idl

```diff
@@ -174,3 +175,21 @@ interface USBEndpoint {
   readonly attribute USBEndpointType type;
   readonly attribute unsigned long packetSize;
 };
+
+dictionary USBPermissionDescriptor : PermissionDescriptor {
+  sequence<USBDeviceFilter> filters;
+};
+
+dictionary AllowedUSBDevice {
+  required octet vendorId;
+  required octet productId;
+  DOMString serialNumber;
+};
+
+dictionary USBPermissionStorage {
+  sequence<AllowedUSBDevice> allowedDevices = [];
+};
+
+interface USBPermissionResult : PermissionStatus {
```

Comment:
I was expecting this to break the test, because PermissionStatus isn't known, but I think what I've seen is that if both interfaces *are* implemented, then one will reach some code path in idlharness.js where if we don't have any information about the right-hand side, the failure is pretty mysterious.

So... I added what I think will be needed for anyone who does implement these bits.